app/controllers/idv/doc_auth_controller.rb

```diff
@@ -54,5 +57,14 @@ def do_meta_refresh(meta_refresh_count)
     def flow_session
       user_session['idv/doc_auth']
     end
+
+    def check_for_outage
+      vendor_status = VendorStatus.new
+      if vendor_status.any_ial2_vendor_outage?
+        session[:vendor_outage_redirect] = current_step
+        session[:vendor_outage_redirect_from_idv] = true
+        return redirect_to vendor_outage_url
+      end
+    end
```

Comment:
minor clean up ideas:
- don't save `vendor_status` variable, it's not used again
- no need for the return, there's nothing else in the method?

```suggestion
    def check_for_outage
      if VendorStatus.new.any_ial2_vendor_outage?
        session[:vendor_outage_redirect] = current_step
        session[:vendor_outage_redirect_from_idv] = true
        redirect_to vendor_outage_url
      end
    end
```